src/fragments/lib/storage/ios/upload.mdx

```diff
@@ -138,6 +138,48 @@ resultSink = uploadTask
 
 </BlockSwitcher>
 
+### Working with Security Scoped Resources (from iCloud)
+Security scoped resources refer to files that are retrieved from iCloud. 
```

Comment:
```suggestion
Security scoped resources refer to files that are retrieved from iCloud or other cloud storage providers. You're likely to run into these file types when using system components that provide access to files stored in iCloud, e.g. [UIDocumentBrowserViewController](https://developer.apple.com/documentation/uikit/uidocumentbrowserviewcontroller).

Reply:
Recommend adding some more context around security scoped resources and when customers may encounter them.

Reply:
Updated per recommendation.